cmd/create/cluster/cmd.go

```diff
@@ -2345,12 +2366,20 @@ func handleByoOidcOptions(r *rosa.Runtime, cmd *cobra.Command, isSTS bool) (bool
 	return isByoOidcSet, oidcEndpointUrl, oidcPrivateKeySecretArn
 }
 
-func minReplicaValidator(multiAZ bool) interactive.Validator {
+func minReplicaValidator(multiAZ bool, isHostedCP bool, privateSubnetsCount int) interactive.Validator {
 	return func(val interface{}) error {
 		minReplicas, err := strconv.Atoi(fmt.Sprintf("%v", val))
 		if err != nil {
 			return err
 		}
+		if isHostedCP {
+			if minReplicas%privateSubnetsCount != 0 {
```

Comment:
Does this risk a panic if privateSubnetsCount == 0?

Reply:
Added an extra guard for it; "privateSubnetsCount" is validated in a previous step, and that case should fail before entering this method, but it's a fair check to add it there too.